google/cloud/spanner_v1/pool.py

```diff
@@ -205,15 +206,16 @@ def bind(self, database):
             session_template=Session(creator_role=self.database_role),
         )
 
-        while not self._sessions.full():
-            resp = api.batch_create_sessions(
-                request=request,
-                metadata=metadata,
-            )
-            for session_pb in resp.session:
-                session = self._new_session()
-                session._session_id = session_pb.name.split("/")[-1]
-                self._sessions.put(session)
+        while trace_call("Cloudspanner.FixedPool.BatchCreateSessions", self):
```

Comment:
1. We cannot pass self to trace_call since the class here is pool not session.
2. we should also pass observability_options here to consider local tracer provider if any set by customer.

Reply:
Done, thanks!